### DISCUSSION TITLE
Custom sorting in Pivot table

### DISCUSSION THREAD

**Discussion by Ajith-mialo** [getredash/redash#6667] [Q&A] [answered]
I'm curious if there's a way to implement custom sorting for rows and columns based on months in Pivot table. The default sorting options for rows and columns include ASC and DESC, but I'm interested in arranging the values of months in their chronological order, rather than the default alphabetical sorting. Is there a way to achieve this through custom code

**Comment by guidopetri** [comment, marked as answer]
You can sort that way with SQL itself, e.g.: [link](https://www.designcise.com/web/tutorial/how-to-custom-sort-in-sql-order-by-clause)
